src/renderer/components/properties-panel/ComputeNodesPropertiesTab.tsx

```diff
@@ -381,12 +382,12 @@ const ComputeNodesPropertiesTab = ({
         <div className={`properties-container ${selectedNodes.length > 0 ? '' : 'empty'}`}>
             <div className='properties-list'>
                 <div className='properties-panel-nodes'>
-                    {selectedNodes.map(({ node, graphName }) => (
+                    {selectedNodes.map((node) => (
                         <ComputeNodePropertiesCard
                             key={node?.uid}
                             node={node}
                             temporalEpoch={epoch}
-                            graphName={graphName}
+                            allLinksState={allLinksState}
```

Comment:
we dont need all link states passed here, we can just grab what we need maybe?

Reply:
Yes, this is not the optimal solution, I'll add a TODO, but maybe fix it on another PR.